lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -39,113 +39,226 @@
+        </a>
+      % endif
+      % if settings.FEATURES.get('ENABLE_VERIFIED_CERTIFICATES'):
+        <span class="sts-enrollment" title="${course_verified_certs.get('enrollment_title')}">
+          <span class="label">${_("Enrolled as: ")}</span>
+          % if course_verified_certs.get('show_image'):
+              <img class="deco-graphic" src="${static.url('images/verified-ribbon.png')}" alt="${course_verified_certs.get('image_alt')}" />
+          % endif
+          <div class="sts-enrollment-value">${course_verified_certs.get('enrollment_value')}</div>
+        </span>
+      % endif
+    </div>
+      <div class="wrapper-course-details">
+        <h3 class="course-title">
           % if show_courseware_link:
-             % if not is_course_blocked:
-            <a href="${course_target}">${course.display_number_with_default | h} ${course.display_name_with_default}</a>
+            % if not is_course_blocked:
+              <a href="${course_target}">${course.display_name_with_default}</a>
             % else:
-            <a class="disable-look">${course.display_number_with_default | h} ${course.display_name_with_default}</a>
+              <a class="disable-look">${course.display_name_with_default}</a>
             % endif
           % else:
-            <span>${course.display_number_with_default | h} ${course.display_name_with_default}</span>
+            <span>${course.display_name_with_default}</span>
           % endif
         </h3>
-      </hgroup>
+        <div class="course-info">
+          <span class="info-university">${get_course_about_section(course, 'university')} - </span>
+          <span class="info-course-id">${course.display_number_with_default | h}</span>
+          <span class="info-date-block" data-tooltip="Hi">
+          % if course.has_ended():
+            ${_("Ended - {end_date}").format(end_date=course.end_datetime_text("SHORT_DATE"))}
+          % elif course.has_started():
+            ${_("Started - {start_date}").format(start_date=course.start_datetime_text("SHORT_DATE"))}
+          % elif course.start_date_is_still_default: # Course start date TBD
+            ${_("Coming Soon")}
+          % else:   # hasn't started yet
+            ${_("Starts - {start_date}").format(start_date=course.start_datetime_text("SHORT_DATE"))}
+          % endif
+          </span>
+        </div>
+        % if show_courseware_link:
+          <div class="wrapper-course-actions">
+            <div class="course-actions">
+              % if course.has_ended():
+               % if not is_course_blocked:
+                <a href="${course_target}" class="enter-course archived">${_('View Archived Course')}</a>
+              % else:
+                <a class="enter-course-blocked archived">${_('View Archived Course')}</a>
+              % endif
```

Comment:
Nitpick: please adjust the indentation on this last `else` block to match the `if` statement above.